ui/src/shared/components/Dygraph.js

```diff
@@ -65,18 +71,26 @@ export default class Dygraph extends Component {
 
     const graphRef = this.graphRef
     const legendRef = this.legendRef
-    let finalLineColors = overrideLineColors
+    const finalLineColors = [...(overrideLineColors || LINE_COLORS)]
+
+    const hashColorDygraphSeries = {}
+    const {length} = finalLineColors
```

Comment:
I don't think that will be necessary. Collisions are okay (for this use case), and creating a hash table is unnecessarily complex, for this use case.